ci-operator/config/openstack-k8s-operators/cinder-operator/OWNERS

```diff
@@ -6,6 +6,7 @@
 
 approvers:
 - abays
+- arxcruz
```

Comment:
We don't edit this file manually after the initial patch and its pulled from target repo, please see the comment above in the same file.

~~~
# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
# Fetched from https://github.com/openstack-k8s-operators/cinder-operator root OWNERS
# If the repo had OWNERS_ALIASES then the aliases were expanded
# Logins who are not members of 'openshift' organization were filtered out
# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
~~~

I think its okay to leave you and robot will remove you, if you want to be added in OWNERS file then you need to push a patch against https://github.com/openstack-k8s-operators/cinder-operator root OWNERS to add yourself.